### PR TITLE
Add ZH BLog: aws-kubernetes

### PR DESCRIPTION
**Pull request by 24sama**
Add a blog.

### CODE REVIEW

File: content/zh/blogs/aws-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+author: '李耀宗'
+snapshot: '../../../images/blogs/aws-kubernetes/aws.png'
+---
+
+### 介绍
+
+对于生产环境，我们需要考虑集群的高可用性。本文教您部署如何在多台 AWS EC2 实例快速部署一套高可用的生产环境。要满足 Kubernetes 集群服务需要做到高可用，需要保证 kube-apiserver 的 HA ，可使用下列两种方式：
+
+* AWS ELB（推荐）
+* keepalived + haproxy [keepalived + haproxy](https://kubesphere.com.cn/forum/d/1566-kubernetes-keepalived-haproxy)对 kube-apiserver 进行负载均衡，实现高可用 Kubernetes 集群。
```
Comment (FeynmanZhou) [inline content/zh/blogs/aws-kubernetes.md:16]: `keepalived + haproxy` is duplicated, please remove the first one.

File: content/zh/blogs/aws-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+keywords: 'Kubernetes, AWS, KubeKey, 高可用, 亚马逊'
+description: 'KubeKey 是一款可以快速、便捷部署高可用 Kubernetes 集群的工具。本文将主要介绍如何在亚马逊 AWS 部署高可用的 Kubernetes 集群。'
+createTime: '2021-08-05'
+author: '李耀宗'
+snapshot: '../../../images/blogs/aws-kubernetes/aws.png'
+---
+
+### 介绍
+
+对于生产环境，我们需要考虑集群的高可用性。本文教您部署如何在多台 AWS EC2 实例快速部署一套高可用的生产环境。要满足 Kubernetes 集群服务需要做到高可用，需要保证 kube-apiserver 的 HA ，可使用下列两种方式：
```
Comment (FeynmanZhou) [inline content/zh/blogs/aws-kubernetes.md:13]: 对于生产环境，我们需要考虑集群的高可用性。
=> 对于生产环境，我们需要考虑 Kubernetes 集群的高可用性。

File: content/zh/blogs/aws-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+在当前位置创建部署配置文件`config-HA.yaml`:
+
+```
+./kk create config -f config-HA.yaml
+```
+
+### 集群配置调整
+
+根据当前集群信息修改配置文件内容，有关更多信息，请参见[示例配置文件](https://github.com/kubesphere/kubekey/blob/release-1.1/docs/config-example.md)：
```
Comment (FeynmanZhou) [inline content/zh/blogs/aws-kubernetes.md:152]: I think we should give a link to the KubeSphere official documentation here.

Comment (24sama) [reply content/zh/blogs/aws-kubernetes.md]: ok, done.

**Comment by 24sama** [comment]
still waiting...

**Review by FeynmanZhou — APPROVED** [review]

**Comment by FeynmanZhou** [comment]
@24sama  At least, it looks better than before. But there are still some places should be followed with the [style guide](https://github.com/kubesphere/website/blob/master/localization_style_guides/KubeSphere%20Localization%20Style%20Guide%20(for%20Simplified%20Chinese).md), you can modify it later. 

/lgtm

**Pull request merged**